### ISSUE TITLE
CSV with blank field

### ISSUE THREAD

**Issue by sdkn104** [ws-garcia/VBA-CSV-interface#9]
VBA CSV interface returns wrong results when importing the following CSV.

```
a,b<CRLF>c,<CRLF>
```
(Two record / two fields CSV. The last field is blank.)

My test code is the following.
```
    Dim MyArray() As String
    Set CSVix = New CSVinterface
    Call CSVix.OpenConnection(fileName)
    Call CSVix.ImportFromCSV
    Call CSVix(MyArray)
    Debug.Print LBound(MyArray, 1)  ' ---> ERROR
```

**Comment by ws-garcia** [comment]
Please, check out the VBA CSV interface new release. Keep in mind, the parser still ignoring lines containing only non-printable chars like spaces, vbTabs and others. Best regards!

**Comment by ws-garcia** [comment]
This issue was closed in [v3.1.0](https://github.com/ws-garcia/VBA-CSV-interface/releases/tag/v3.1.0). Please, check it!
